### ISSUE TITLE
it scans really slow, anyway to speed it up? 

### ISSUE THREAD

**Issue by Antonijow** [AHAAAAAAA/PokemonGo-Map#2591]
it feels like i am missing alot of pokemon beacuse it scans so slow
i am using st 5

**Comment by joshboldman** [comment]
Same here

**Comment by TFD777** [comment]
Add EXTRA_ARGS key with value -sd 0

**Comment by JoshuaGarrison27** [comment]
Just to give context to @TFD777 's answer, `-sd` is the step delay. (see more here: [Command Arguments](https://github.com/AHAAAAAAA/PokemonGo-Map/wiki/Command-Line-Arguments))

The default value for this is '5'. See the code here... [https://github.com/AHAAAAAAA/PokemonGo-Map/blob/master/pogom/utils.py](https://github.com/AHAAAAAAA/PokemonGo-Map/blob/master/pogom/utils.py)

**Comment by Antonijow** [comment]
oh yeah that really works, it is searching alot faster
although now i get a error every 3 lines saying searh thread failed, response dictionary key error

**Comment by tarthim** [comment]
@Antonijow Yea, Niantic is blocking requests is they come too quickly at the moment.

**Comment by Pym** [comment]
@Antonijow try with `-sd 0.15` ;)

**Comment by xomikronx** [comment]
@tarthim do you know if this is a temporary or contingency method to help control load to servers or is it a permanent throttle?  I'm not getting any errors with 3 threads, but anything beyond give:
`Niantic servers under heavy load. Waiting before trying again`

**Comment by tarthim** [comment]
@Pym Does fail quite a bit, doubt it's really that much faster than 1

**Comment by tarthim** [comment]
@xomikronx Don't think anyone knows atm except for Niantic, but I /think/ it's pretty much a permanent thing :/

**Comment by talentedmrripley** [comment]
With a bit of testing and some info from other reports the best I can get is -t 3 and -sd 1.1.  That works pretty fast and doesn't generate any errors.

**Comment by realbunny** [comment]
@talentedmrripley 
Doesn't look like it accurate though, I'm testing it with pokevision and some pokemon are missing

**Comment by talentedmrripley** [comment]
@realbunny Are you seeing errors in the output?  Missing Pokemon is a different issue than a slow scan potentially caused by connection throttling.

**Comment by realbunny** [comment]
@talentedmrripley no, im not. But i know what the problem is, but dont know where to fix it. There was a post saying about that each scan is now 70m, not 100m. But i dont know where to fix that.

**Comment by moggd** [comment]
@realbunny 
Go to pogom/search.py

Change pulse_radius to 0.07.  This will fix the 100m > 70m bug
